### PR TITLE
Confirm the default number of primary shards for application and operations indices equals data node count

### PR DESCRIPTION
**Pull request by mburke5678**
https://jira.coreos.com/browse/OSDOCS-688

### CODE REVIEW

**Comment by mburke5678** [comment]
@ewolinetz PTAL.
Does this Jira (https://jira.coreos.com/browse/OSDOCS-688) need any more mention in the docs than a note?

File: modules/efk-logging-elasticsearch-ha.adoc [hunk 1]
```diff
@@ -20,6 +20,10 @@ Logs might be unavailable or lost in the event a node is down or fails.
 Use this mode when you are more concerned with performance than safety, or have
 implemented your own disk/PVC backup/restore strategy.
 
+[NOTE]
+====
+The number of primary shards is always equals to the number of Elasticsearch nodes.
```
Comment (ewolinetz) [inline modules/efk-logging-elasticsearch-ha.adoc]: I think we want to be clear that this is specifically for our index templates. The number of primary shards on an index is set during creation and cannot be changed after that... also specifically data nodes.

"The number of primary shards for the index templates is equal to the number of Elasticsearch data nodes".

**Comment by mburke5678** [comment]
@anpingli Please take a look.

**Review by anpingli — COMMENTED** [review]
LGTM

**Review by kalexand-rh — COMMENTED** [review]
LGTM!

**Pull request merged**